tools/device_flow/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "kanidm_device_flow"
+default-run = "kanidm_device_flow"
+description = "Kanidm Device Flow Client"
+documentation = "https://kanidm.github.io/kanidm/stable/"
+version = { workspace = true }
+authors = { workspace = true }
+rust-version = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+
+[features]
+
+[[bin]]
+name = "kanidm_device_flow"
+path = "src/main.rs"
+
+[dependencies]
+kanidm_proto = { workspace = true }
+anyhow = { workspace = true }
+oauth2 = "4.4.2"
+reqwest = { version = "0.12.8", default-features = false, features = [
```

Comment:
If you have tokio, why do you need blocking?

Reply:
Because that's what the oauth2 client uses for its "simple get things working" example code, and I'm only half done writing this :D